docs/source/bird-rr-config.rst

```diff
@@ -0,0 +1,149 @@
+Configuring BIRD as a BGP Route Reflector
+=========================================
+
+These instructions will take you through installing BIRD as a BGP route
+reflector, and updating your other BIRD instances to speak to your new route
+reflector.
+
+These instructions are valid for both Ubuntu 14.04 and RHEL 7.
+
+Prerequisites
+-------------
+
+Before starting this you will need the following:
+
+- A machine running either Ubuntu 14.04 or RHEL 7.
+- SSH access to the machine.
+
+Installation
+------------
+
+Step 1: Install BIRD
+~~~~~~~~~~~~~~~~~~~~
+
+Ubuntu 14.04
+^^^^^^^^^^^^
+
+Add the official `BIRD`_ PPA. This PPA contains fixes to BIRD that are not yet
+available in Ubuntu 14.04. To add the PPA, run::
+
+    sudo add-apt-repository ppa:cz.nic-labs/bird
+
+Once that's done, update your package manager and install BIRD::
+
+    sudo apt-get update
+    sudo apt-get install bird
+
+
+.. _BIRD: http://bird.network.cz/
+
+RHEL 7
+^^^^^^
+
+First, install EPEL. Depending on your system, the following command may be
+sufficient::
+
+    sudo yum install epel-release
+
+If that fails, try the following instead::
+
+    wget https://dl.fedoraproject.org/pub/epel/7/x86_64/e/epel-release-7-5.noarch.rpm
+    sudo yum install epel-release-7-5.noarch.rpm
+
+With that complete, you can now install BIRD::
+
+    yum install -y bird bird6
+
+Step 2: Set your BIRD IPv4 configuration
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Before doing this, you'll need to take note of what BGP AS number you've used
+in your compute node install.
```

Comment:
Where would this be found - can we give people a pointer?  [I don't know, for example, whether in the "normal" Calico install we tell people to explicitly configure this, or just assume they'll leave it as a default value.]